modules/@ergonode/core/src/models/mappers/errorsMapper.js

```diff
@@ -16,13 +17,22 @@ export const getMappedErrors = ({
         const fieldKey = fieldKeys[key] || key;
 
         if (Array.isArray(errors[key])) {
-            mappedErrors[fieldKey] = errors[key].join(', ');
+            if (getOnlyFirstError) {
+                const [
+                    errorMessage,
+                ] = errors[key];
```

Comment:
You are repeating `errors[key]` few times in this function - you can assign it to `const` and reuse it.